### ISSUE TITLE
TypeError when saving stream from web interface

### ISSUE THREAD

**Issue by kmahelona** [LibreTime/libretime#968]
**Describe the bug**
Saving a stream from the web UI `/preference/stream-setting` raises a TypeError. This is likely a `python3` error (see below for my configuration).

**To Reproduce**
1. Go to `/preference/stream-setting`
2. Click on `save`
3. Liquidsoap is still streaming but no sound is played

**Expected behavior**
We should still hear playout.

**LibreTime version:**
My LibreTime is a mix of a few PRs:
- python3 #951 
- watch folder #908 
- Ubuntu Eoan #967

It seems most probably a python3 error but I will investigate further.

**Installation method:**
 - OS: Ubuntu
 - OS Version: 19.10
 - Method: `./install`
 - Details: Deployed with https://github.com/TeHikuMedia/libretime-deploy.

**Error***
`TypeError: 'in <string>' requires string as left operand, not bytes`

**Logs**
```
2020-03-04 10:09:59,826 [pypofetch] [INFO ]  Loop #1
2020-03-04 10:11:03,017 [pypoliqqueue] [INFO ]  waiting 244.982688s until next scheduled item
2020-03-04 10:14:57,564 [pypomessagehandler] [INFO ]  Received event from RabbitMQ: {"stream_format":"0","event_type":"update_stream_format"}
2020-03-04 10:14:57,564 [pypomessagehandler] [INFO ]  Handling command: update_stream_format
2020-03-04 10:14:57,564 [pypomessagehandler] [INFO ]  Updating stream format...
2020-03-04 10:14:57,565 [pypofetch] [INFO ]  Received event from Pypo Message Handler: {"stream_format":"0","event_type":"update_stream_format"}
2020-03-04 10:14:57,565 [pypofetch] [INFO ]  Handling command: update_stream_format
2020-03-04 10:14:57,565 [pypofetch] [INFO ]  Updating stream format...
2020-03-04 10:14:57,566 [pypofetch] [INFO ]  b'vars.stream_metadata_type 0\n'
2020-03-04 10:14:57,566 [pypofetch] [ERROR]  'in <string>' requires string as left operand, not bytes
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/airtime_playout-1.0-py3.7.egg/pypo/pypofetch.py", line 267, in update_liquidsoap_stream_format
    tn.write('exit\n')
  File "/usr/lib/python3.7/telnetlib.py", line 287, in write
    if IAC in buffer:
TypeError: 'in <string>' requires string as left operand, not bytes
2020-03-04 10:14:57,569 [pypofetch] [INFO ]  New timeout: 182.1806378364563
2020-03-04 10:14:57,569 [pypofetch] [INFO ]  Loop #2
```

**Comment by kmahelona** [comment]
This particular error is resolved in 9ef09a7, but I'm still having to systematically stop playout, stop liquidsoap, then start playout, then start liquidsoap to get my audio stream to work again.

**Comment by kmahelona** [comment]
The errors mentioned above are all caused by the move to python3. They have been resolved in #951 by https://github.com/paddatrapper/libretime/pull/1
